src/plugins/process/testapp.sh

```diff
@@ -74,7 +74,7 @@ system:/elektra/modules/process/infos/recommends
 
 $key string 44 42
```

Comment:
This script prints `dump` format, so when you change the value of the key, you also need to update it's length accordingly:

```suggestion
$key string 44 48
```

@Janldeboer That should fix the issues with `testmod_process`

Reply:
Thank you very much! I don't think I would have ever thought of that